src/components/OpacityView.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Animated, {AnimatedStyle, useAnimatedStyle, useSharedValue, withTiming} from 'react-native-reanimated';
+import {StyleProp, ViewStyle} from 'react-native';
+import variables from '../styles/variables';
+import shouldRenderOffscreen from '../libs/shouldRenderOffscreen';
+
+type OpacityViewProps = {
+    /**
+     * Should we dim the view
+     */
+    shouldDim: boolean;
+
+    /**
+     * Content to render
+     */
+    children: React.ReactNode;
+
+    /**
+     * Array of style objects
+     * @default []
+     */
+    style: StyleProp<AnimatedStyle<ViewStyle>>;
```

Comment:
```suggestion
    style?: StyleProp<AnimatedStyle<ViewStyle>>;
```

You forgot to leave it optional in last review 😅

Reply:
Done! @fabioh8010